indico/modules/events/sessions/forms.py

```diff
@@ -40,3 +41,17 @@ class SessionForm(IndicoForm):
                                       description=_('Specify text and background colours for the session.'))
     is_poster = BooleanField(_('Poster session'), widget=SwitchWidget(),
                              description=_('Whether the session is a poster session.'))
+
+
+class EmailSessionPersonsForm(IndicoForm):
+    from_address = SelectField(_('From'), [DataRequired()], choices=[(1, 1)])
+    subject = StringField(_('Subject'), [DataRequired()])
+    body = TextAreaField(_('Email body'), [DataRequired()], widget=CKEditorWidget(simple=True))
+    event_persons = JSONField(_('Event Persons'), default=[])
+
+    def __init__(self, *args, **kwargs):
+        self.event = kwargs.pop('event')
```

Comment:
since you only use it in here you don't have to store it on `self`. having a local `event` variable is enough